Initialtask/src/Command/ProductsImportCommand.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\FileParser;
+use App\Entity\Product;
+use App\Repository\FileParserRepository;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Doctrine\ORM\EntityManagerInterface;
+
+
+class ProductsImportCommand extends Command
+{
+    protected static $defaultName = 'products:import';
```

Comment:
Please use command as a service, move name of the command into yml service definition